packages/simcore-sdk/src/simcore_sdk/node_ports_common/aws_s3_cli_utils.py

```diff
@@ -13,7 +13,9 @@
 def _parse_size(log_string):
     match = re.search(r"^\w+ (?P<size>[^\/]+)", log_string)
     if match:
-        return match.group("size")
+        # NOTE: ByteSize does not know what `Bytes` are.
+        # It only knows about `b` and omitting the word bytes if they are just bytes.
+        return match.group("size").replace("Bytes", "")
```

Comment:
`1 Byte` (singular) could be the only case uncovered

Reply:
perhaps you should add a test ... to be 100% sure